ajax/libs/jodit/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jodit",
-  "version": "3.2.65",
+  "version": "3.3.2",
```

This change needs to be reverted please
```suggestion
  "version": "3.2.65",
```